src/app/backend/resource/networkpolicy/common.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2017 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package networkpolicy
+
+import (
+	"github.com/kubernetes/dashboard/src/app/backend/api"
+	metricapi "github.com/kubernetes/dashboard/src/app/backend/integration/metric/api"
+	"github.com/kubernetes/dashboard/src/app/backend/resource/dataselect"
+	networkpolicy "k8s.io/api/networking/v1"
+	"encoding/json"
+	"log"
+)
+
+func toNetworkPolicy(networkpolicy *networkpolicy.NetworkPolicy) NetworkPolicy {
+	//deepcopy
+	log.Println("marshl networkpolicy ",networkpolicy)
+	byte,err:=json.Marshal(networkpolicy.Spec)
+	log.Println("marshl networkpolicy spec",string(byte))
```

Comment:
Too many logs.

Reply:
very sorry,i debug this module  many times ,forget to delete the log